.github/workflows/main.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           - { rust: stable, target: i686-pc-windows-msvc, os: windows-latest }
           - { rust: beta, target: x86_64-unknown-linux-gnu, os: ubuntu-latest }
           - {
-              rust: nightly,
+              rust: nightly-2022-11-10,
```

Comment:
why test on nightly?

Reply:
to catch issues that we'll run into when nightly becomes stable.

Reply:
also, things like sanitizers only work on nightly